static/src/javascripts/projects/common/modules/article/chapters.js

```diff
@@ -0,0 +1,24 @@
+define([
+    'bean',
+    'common/utils/$'
+], function (
+    bean,
+    $
+) {
+    return {
+        init: function () {
+            this.bindEvents();
+        },
+
+        bindEvents: function () {
+            bean.on(document.body, 'click', '.js-toggle-chapters', function (e) {
+                e.preventDefault();
```

Comment:
Do we need to preventDefault here? Can't we just remove the href from the button?